src/components/partnership/dto/update-partnership.dto.ts

```diff
@@ -32,6 +33,9 @@ export abstract class UpdatePartnership {
 
   @Field(() => [PartnershipType], { nullable: true })
   readonly types?: PartnershipType[];
+
+  @Field(() => PartnershipFundingType, { nullable: true })
+  readonly fundingType?: PartnershipFundingType;
```

Comment:
```suggestion
  readonly fundingType?: PartnershipFundingType | null;
```
This would fix the need to typecast

Reply:
If I change like above, there's type error in `sgUpdateProperties`
`Type 'PartnershipFundingType | null | undefined' is not assignable to type 'PartnershipFundingType | undefined'`

Reply:
Ah! Ok yeah this is because we are ambiguous with our output types on whether null is allowed when you do have permission to read it. Let me try to fix. I know we've been back and forth on this, but these are good things that we've been putting off.

Reply:
Okay, thanks

Reply:
Ok I've got https://github.com/SeedCompany/cord-api-v3/pull/975 up. You can rebase and `{ nullable: true }` to `SecuredPartnershipFundingType` after it's merged.

Reply:
Okay